### PR TITLE
[ios] Use default mail app on iOS 14 to report a bug

### LINKED ISSUE #3281: [iOS] "Report a bug" doesn't jump to default mail app
The "report a bug" link jumps to outlook app on iPhone and gmail app on iPad, even though the default mail app on both devices is the Mail app.

OM Version: ⁦2022.08.23-3⁩

### PR DESCRIPTION
**Pull request by biodranik**
Fixes #3281

It would be great to test it first )

### CODE REVIEW

**Comment by TheAdventurer64** [comment]
I can theyy and test this later today if I can

**Review by vng — APPROVED** [review]

File: iphone/Maps/UI/Help/AboutController.swift [hunk 1]
```diff
@@ -256,12 +256,12 @@ final class AboutController: MWMViewController, UITableViewDataSource, UITableVi
     let subject = emailSubject(subject: header)
     let body = emailBody()
 
-    // Try Gmail and Outlook first.
-    if openGmail(subject: subject, body: body, recipients: toRecipients)
-        || openOutlook(subject: subject, body: body, recipients: toRecipients) {
+    // Before iOS 14, try to open alternate email apps first, assuming that if users installed them, they're using them.
+    let os = ProcessInfo().operatingSystemVersion
+    if (os.majorVersion < 14 && (openGmail(subject: subject, body: body, recipients: toRecipients) || openOutlook(subject: subject, body: body, recipients: toRecipients))) {
```
Comment (vng) [inline iphone/Maps/UI/Help/AboutController.swift:261]: Тут уже совсем длинное выражение получилось ..
Давайте понятийно придерживаться +- 130 символов, чтобы хотя бы в github не скролить.

**Review by rtsisyk — APPROVED** [review]

**Comment by biodranik** [comment]
Did anyone test it? :)

**Comment by vng** [comment]
Suppose that you did some simple test here :)

**Comment by biodranik** [comment]
Nope ) I wrote it in the PR comment )

**Comment by vng** [comment]
So why you did not test your changes? :)

**Comment by biodranik** [comment]
Because I don't have other mail apps.

**Pull request merged**